tests/integration/image_upload.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use File::Basename "dirname";
+use File::Path qw/make_path remove_tree/;
+use Getopt::Long qw/GetOptions/;
+use Log::Any::Adapter 'TAP';
+use Mock::Quick qw/qobj qmeth/;
+
+my $test_dir = dirname(__FILE__);
+my $inputs_dir = "$test_dir/inputs/image_upload/";
+
+my $update_expected_results;
+
+GetOptions("update-expected-results" => \$update_expected_results)
+  or die("Error in command line arguments.\n\n");
+
+remove_all_products();
+wait_dynamic_front();
+
+my $admin_ua = new_client();
+create_user($admin_ua, {});
+
+my %product_fields = (
+	code => '200000000099',
+	lang => "en",
+	product_name => "Testttt-75ml",
+	generic_name => "Tester",
+	quantity => "75 ml",
+	link => "https://github.com/openfoodfacts/openfoodfacts-server",
+	expiration_date => "test",
+	ingredients_text => "apple, milk",
+	origin => "france",
+	serving_size => "10g",
+	packaging_text => "no",
+	action => "process",
+	type => "add",
+	".submit" => "submit"
+);
+
+create_product($admin_ua, \%product_fields);
+
+my $response = $admin_ua->post("http://world.openfoodfacts.localhost/cgi/product_jqm2.pl?code=200000000099&product_image_upload.pl/imgupload_front=$inputs_dir/apple.jpg", Content => "$inputs_dir/apple.jpg");
```

Comment:
maybe something like (not sure)

```suggestion
my $image = open("$inputs_dir/apple.jpg", "r") # with right syntax ;-)
my $response = $admin_ua->post(
  "http://world.openfoodfacts.localhost/cgi/product_image_upload.pl, 
  [$image, "apple.jpg", code => "200000000099", imagefield => "front"]);
```

We would need to be sure because documentation seems strange.

Reply:
There's an example in https://metacpan.org/pod/HTTP::Request::Common


```

POST 'http://www.perl.org/survey.cgi',     Content_Type => 'form-data',     Content      => [ name  => 'Gisle Aas',                       email => 'gisle@aas.no',                       gender => 'M',                       born   => '1964',                       init   => ["$ENV{HOME}/.profile"],                     ]
--

```

Reply:
So something like:

```
my $response = $admin_ua->request(POST 
	  "http://world.openfoodfacts.localhost/cgi/product_image_upload.pl, 
 Content_Type => 'form-data',     
Content      => [ 
code => "200000000099"
imagefield => "front",
imgupload_front => ["$inputs_dir/apple.jpg],
]
);

```

Reply:
tried this out